src/components/scenes/DuressModeSettingScene.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react'
+import { useEffect } from 'react'
+import { Image, View } from 'react-native'
+
+import illustrationImageSource from '../../assets/images/duressMode/illustration.png'
+import { useHandler } from '../../hooks/useHandler'
+import { useWatch } from '../../hooks/useWatch'
+import { lstrings } from '../../locales/strings'
+import { useSelector } from '../../types/reactRedux'
+import { EdgeAppSceneProps } from '../../types/routerTypes'
+import { ButtonsView } from '../buttons/ButtonsView'
+import { EdgeCard } from '../cards/EdgeCard'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { styled } from '../hoc/styled'
+import { showError, showToast } from '../services/AirshipInstance'
+import { EdgeText, Paragraph } from '../themed/EdgeText'
+import { onPin } from './DuressPinScene'
+
+interface Props extends EdgeAppSceneProps<'duressModeSetting'> {}
+
+export const DuressModeSettingScene = (props: Props) => {
+  const { navigation } = props
+  const account = useSelector(state => state.core.account)
+
+  const handleDisableDuress = useHandler(async () => {
+    await account.changePin({ enableLogin: false, forDuressAccount: true })
+  })
+
+  const handleEnableDuress = useHandler(async () => {
+    navigation.navigate('duressPin')
+  })
+
+  const handleLearnMore = useHandler(() => {
+    navigation.navigate('duressModeHowTo')
+  })
+
+  const isDuressModeConfigured = useWatch(account, 'isDuressAccountEnabled')
+
+  // Subscribe to pin event from VerifyPinScene
+  useEffect(
+    () =>
+      onPin(pin => {
+        account
+          .checkPin(pin)
+          .then(async isCorrect => {
+            if (isCorrect) {
+              showToast(lstrings.duress_mode_pin_match_rule_message)
+              return
+            }
+            await account.changePin({ pin, forDuressAccount: true })
+            // Navigate back to this scene
+            navigation.navigate('duressModeSetting')
+          })
+          .catch(err => showError(err))
+      }),
+    [account, navigation]
+  )
+
+  return (
+    <SceneWrapper scroll>
+      <HeadingContainer>
+        <HeadingImageContainer>
+          <HeadingImage source={illustrationImageSource} />
+        </HeadingImageContainer>
+        <Heading>{lstrings.title_duress_mode}</Heading>
+      </HeadingContainer>
+
+      <BodyContainer>
```

Comment:
Same comment as `DuressModeHowToScene` - we should just have one view that
manages the entire outer expected margins on scene edges, since all the inner components (with
a few exceptions such as scrolling containers) all expect 0.5rem outer margins

Reply:
Removed the BodyContainer here as well in favor of SceneContainer